lib/nimble_template/templates/template.ex

```diff
@@ -52,6 +56,37 @@ defmodule NimbleTemplate.Templates.Template do
     Mix.shell().cmd("npm install --prefix assets")
   end
 
+  defp suppress_necessary_credo_warnings(%Project{
+         base_module: base_module,
+         base_path: base_path,
+         web_path: web_path,
+         web_test_path: web_test_path
+       }) do
+    Enum.each(
+      [
+        "lib/#{Macro.underscore(base_module)}.ex",
+        "#{web_path}/controllers/page_controller.ex",
+        "#{web_path}/telemetry.ex",
+        "#{web_path}/views/error_view.ex",
+        "#{web_path}/views/error_view.ex",
+        "#{base_path}/release_tasks.ex"
+      ],
+      fn file_path ->
+        CredoHelper.disable_rule(file_path, "CompassCredoPlugin.Check.DoSingleExpression")
```

Comment:
IMO, we should disable the rule for the entire file instead of fixing these file content because these files are generated from the Phoenix generator, and this template can be used with different Phoenix generator versions depending on the user's machine. The different generator versions can result in different file content, so we should disable the rule for the file.

What do you think? 💭

Reply:
I'm not sure @hanam1ni as we would like to use the template to generate the project and we should reduce the `manual changes` as much as we can on the `output project` so with the `disable credo rule` like this, later on the people will need to remove it and adjust the file manually 🤔

Reply:
@andyduong1920 Some files that are generated with Phoenix generator are not comply with our Compass convention, such as some of the functions in `telemetry.ex` or `release_tasks.ex` does not follow our convention [here](https://nimblehq.co/compass/development/code-conventions/elixir/#convention-20). 
```elixir
# telemetry.ex
defmodule ProjectWeb.Telemetry do
  def start_link(arg) do
    Supervisor.start_link(__MODULE__, arg, name: __MODULE__)
  end

  # ...
end
```
So we need to disable rules in some generated files, which we don't have control over them, to make prevent Credo raise any warning from the fresh generated project and also to pass the CI.